modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,70 +1,75 @@
 @import 'atlas-variables';
+@import 'cadmin-variables';
 @import 'mixins';
```

Comment:
I don't see any custom mixins in this module. Importing `cadmin-variables` or `atlas-variables` imports Clay CSS mixins and functions for us.